File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BRIDGE/IniciaVendaDebito.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE

import com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.BridgeCommand

class IniciaVendaDebito(
    private val idTransacao: Int,
    private val pdv: String,
    private val valorTotal: String
) : BridgeCommand("IniciaVendaDebito") {
    override fun functionParameters(): String {
        return "\"idTransacao\"" + ":" + idTransacao + "," +
                "\"pdv\"" + ":" + "\"" + pdv + "\"" + "," +
                "\"valorTotal\"" + ":" + "\"" + valorTotal + "\""
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BRIDGE/inicia_venda_debito.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BRIDGE/bridge_command.dart';

class IniciaVendaDebito extends BridgeCommand {
  final int _idTransacao;
  final String _pdv;
  final String _valorTotal;

  IniciaVendaDebito(this._idTransacao, this._pdv, this._valorTotal)
      : super('IniciaVendaDebito');

  @override
  functionParameters() {
    return '"idTransacao"'
        ':'
        '$_idTransacao'
        ','
        '"pdv"'
        ':'
        '"$_pdv"'
        ','
        '"valorTotal"'
        ':'
        '"$_valorTotal"';
  }
}
